xarray/tests/test_backends.py

```diff
@@ -1288,6 +1313,35 @@ def test_roundtrip_string_encoded_characters(self):
     def test_dataset_caching(self):
         super(CFEncodedDataTest, self).test_dataset_caching()
 
+    @pytest.mark.xfail(reason="Zarr stores can not be appended to")
+    def test_append_write(self):
+        super(CFEncodedDataTest, self).test_append_write()
+
+    @pytest.mark.xfail(reason="Zarr stores can not be appended to")
+    def test_append_overwrite_values(self):
+        super(CFEncodedDataTest, self).test_append_overwrite_values()
+
+    @pytest.mark.xfail(reason="Zarr stores can not be appended to")
+    def test_append_with_invalid_dim_raises(self):
+        super(CFEncodedDataTest, self).test_append_with_invalid_dim_raises()
+
+    # zero-dim variables
+    @pytest.mark.xfail(reason="Zero-dimension variables are broken")
+    def test_zero_dimensional_variable(self):
+        super(CFEncodedDataTest, self).test_zero_dimensional_variable()
+
+    @pytest.mark.xfail(reason="Zero-dimension variables are broken")
+    def test_roundtrip_timedelta_data(self):
+        super(CFEncodedDataTest, self).test_roundtrip_timedelta_data()
+
+    @pytest.mark.xfail(reason="Zero-dimension variables are broken")
+    def test_roundtrip_datetime_data(self):
+        super(CFEncodedDataTest, self).test_roundtrip_datetime_data()
+
+    @pytest.mark.xfail(reason="Zero-dimension variables are broken")
+    def test_roundtrip_coordinates_with_space(self):
+        super(CFEncodedDataTest, self).test_roundtrip_coordinates_with_space()
```

Comment:
@rabernat - I have not been able to get zero-dim variables to work with zarr. I know you iterated on this but did you ever get it to work?

xref: https://github.com/zarr-developers/zarr/issues/150

Reply:
This issue was fixed in  zarr-developers/zarr#160. But zarr has not had a release since then. So you need to be installing zarr from github master in order to get zero-dim variables.

Zero dim variables are covered by the test suite
https://github.com/pydata/xarray/blob/master/xarray/tests/test_backends.py#L170
And you can see the test passing in the `zarr-dev` environment in my original zarr PR:
https://travis-ci.org/pydata/xarray/jobs/315571803#L1558

Reply:
you're right. I thought I had Zarr-master in my environment. but I didn't These all work now.